src/compnonents/stats/LineChart.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+	import Chart, { type ChartItem } from 'chart.js/auto';
+	import { chartConfig } from 'utils/chartConfig';
+	import { prefersDarkMode } from 'utils/stores';
+	import { mounted } from 'utils/mounted';
+	import { browser } from '$app/environment';
+
+	export let data: any[];
```

Comment:
Please use a more specific type, eg.
```suggestion
	export let data: { [key: string]: string | number | null }[];
```